src/ServiceBus.AttachmentPlugin.Tests/DocoUpdater.cs

```diff
@@ -1,11 +0,0 @@
-﻿using MarkdownSnippets;
-using Xunit;
-
-public class DocoUpdater
-{
-    [Fact]
-    public void Run()
-    {
-        DirectoryMarkdownProcessor.RunForFilePath();
```

Comment:
I'm not sure I like this option. It was very easy to run this test as a part of the test suite.
And I got it working (https://github.com/SimonCropp/MarkdownSnippets/issues/66#issuecomment-520612687).